terraform/iam.tf

```diff
@@ -21,8 +21,7 @@ module "pipeline_root_bucket_iam" {
   bucket_name = module.gcs_buckets["pipeline_root_bucket"].name
   member      = "serviceAccount:${module.service_accounts["pipelines_sa"].email}"
   roles = [
-    "roles/storage.objectViewer",
-    "roles/storage.objectCreator",
+    "roles/storage.admin" # Added by JAN, this trumps the reason above by giving even higher privileges as because get mysterious AttributeError. See post https://www.mail-archive.com/issues@beam.apache.org/msg183103.html. Example Pipeline: https://console.cloud.google.com/vertex-ai/locations/europe-west4/pipelines/runs/xgboost-train-pipeline-20220912123450?project=dt-jan-sandbox-dev
```

Comment:
No need to say who made this change, we can look in git logs for that. Keep the comment describing why we need `storage.admin`, though.

As this is a PR to the open source template, I'd recommend removing the link to your dt project because users outside dt won't have access to that

Reply:
👍